assets/javascript/developers.js

```diff
@@ -1,25 +1,43 @@
-function Developer(name, price, basePerformance, description, buffs, imageName) {
+function Developer(id, name, price, basePerformance, description, buffs) {
+  this.id = id;
   this.name = name;
-  this.price = price;
+  this.basePrice = price;
   this.basePerformance = basePerformance;
   this.description = description;
   this.buffs = buffs;
   this.level = 0;
   this.speed = 1;
-  this.imageName = imageName;
+
   this.performance = function() {
     let performance = basePerformance + Math.floor((this.level * basePerformance) / 2);
     let extra_performance = 0;
-    this.buffs.forEach(
-      function (buff) {
-        if(buff.active && buff.target === 'performance'){
+
+    this.buffs.forEach(buff => {
+        if(buff.active && buff.target === 'performance') {
           performance += performance * buff.value;
         }
       }
     );
-    performance += extra_performance;
-    return performance
-  }
+
+    return performance += extra_performance;
+  };
+
+  this.levelUp = function (amount) {
+    let level = this.level + amount;
+
+    this.level = level;
+    this.buffs.forEach(
+      function (buff) {
+        if (!buff.active && buff.requiredLevel <= level) {
+          buff.active = true;
+        }
+      }
+    )
+  };
+
+  this.price = function () {
+    return this.basePrice + this.basePrice * this.level * this.level
```

Comment:
If you body is just your return u can use arrow functions to returns it directly 
```js
  this.price = () =>
    this.basePrice + this.basePrice * this.level * this.level;
```